Formula/lib/libnsl.rb

```diff
@@ -13,18 +13,30 @@ class Libnsl < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "fa7613b30e9bfe15166339d119c19115ec21f13cea259280182e0c083502ff40"
   end
 
+  keg_only :shadowed_by_macos
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext" => :build
```

Comment:
This is causing a circular dependency. It's probably better to patch `configure` directly so that we don't need to regenerate it.